Transport/Connection.php

```diff
@@ -510,6 +515,11 @@ static function (): bool {
                     }
                 );
             }
+
+            $this->lastActivityTime = time();
+        } else if (time() > $this->lastActivityTime + ($this->connectionOptions['heartbeat']*2)) {
```

Comment:
Here a explaination to justify the condition : 
https://www.rabbitmq.com/heartbeats.html#:~:text=Heartbeat%20frames%20are%20sent%20about,TCP%20connection%20will%20be%20closed.